x-pack/plugins/observability_solution/apm/public/components/shared/entity_enablement/index.tsx

```diff
@@ -82,7 +88,11 @@ export function EntityEnablement() {
   ) : (
     <EuiFlexGroup direction="row" alignItems="center" gutterSize="xs">
       <EuiFlexItem grow={false}>
-        {isLoading ? <EuiLoadingSpinner size="m" /> : <TechnicalPreviewBadge icon="beaker" />}
+        {isLoading ? (
+          <EuiLoadingSpinner size="m" />
+        ) : (
+          <TechnicalPreviewBadge icon="beaker" style={{ verticalAlign: 'middle' }} />
+        )}
       </EuiFlexItem>
       <EuiFlexItem grow={false}>
         <EuiLink
```

Comment:
Can you disable the link when `isLoading` is true? The action takes quite some time to finish, users might click on it again.

Reply:
good catch, fixed it

https://github.com/user-attachments/assets/b03f3bc3-5f96-485e-8c2b-135f34e07ea3

Reply:
Is it also disabled on the "no data" page?

Reply:
yes

https://github.com/user-attachments/assets/0ec9420d-4a7a-4b03-a017-a2eea37b0627